tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -14,18 +14,18 @@
 class Homescreen(Base):
 
     name = 'Default Home Screen'
-    manifest_url = '{}verticalhome{}/manifest.webapp'.format(Base.DEFAULT_PROTOCOL,Base.DEFAULT_APP_HOSTNAME)
 
     _homescreen_icon_locator = (By.CSS_SELECTOR, 'gaia-grid .icon')
     _homescreen_all_icons_locator = (By.CSS_SELECTOR, 'gaia-grid .icon:not(.placeholder)')
     _homescreen_collection_icon = (By.CSS_SELECTOR, 'gaia-grid .collection')
     _edit_mode_locator = (By.CSS_SELECTOR, 'body.edit-mode')
     _search_bar_icon_locator = (By.ID, 'search-input')
-    _landing_page_locator = (By.ID, 'icons')
+    _landing_page_locator = (By.ID, 'apps')
     _bookmark_icons_locator = (By.CSS_SELECTOR, 'gaia-grid .bookmark')
     _divider_locator = (By.CSS_SELECTOR, 'section.divider')
     _divider_separator_locator = (By.CSS_SELECTOR, 'section.divider .separator > span')
     _exit_edit_mode_locator = (By.ID, 'exit-edit-mode')
+    _icons_locator = (By.TAG_NAME, 'gaia-app-icon')
```

Comment:
Can you add a space before this new locator? I suspect the older locators can be removed for a large part, at some point.